packages-internal/scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/internal-scripts",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
This should be released from `next`, right? https://github.com/mui/material-ui/pull/42898/files#diff-7f1ac39ea5570dd1f7aa1133e4740529dd2eceee2043abc468e242efa004d1f0

Reply:
I think so. Reverted.